.travis-build.sh

```diff
@@ -19,6 +19,16 @@ if [ $USER != "travis" ]; then
     exit 1
 fi
 
+if ! $TRAVIS_SECURE_ENV_VARS && [ $TRAVIS_BRANCH = "master" ]; then
+    printf '=%.0s' {1..70}
+    printf "\n       し(*･∀･)／   Thanks for the contribution!  ＼(･∀･*)ノ\n"
+    printf '=%.0s' {1..70}
+    printf "\n( ＾◡＾)っ Please submit your pull request against the develop branch.\n"
+    echo   "You can read the contribution guidelines at:"
+    echo   "https://github.com/syl20bnr/spacemacs/blob/develop/CONTRIBUTING.org"
+    exit 1
+fi
+
```

Comment:
@robbyoconnor  I think if a build was triggered through the Travis API  then the `$TRAVIS_SECURE_ENV_VARS` should be true.  For the local testing it [should fail anyway](https://github.com/JAremko/spacemacs-pr/blob/7596943a3ab063354a5c88b14665ae7023c76a4a/.travis-build.sh#L16)